data-raw/roman_population.R

```diff
@@ -0,0 +1,5 @@
+# Roman population size estimates
+roman_population <- tibble::tibble(period = c("200 BC:100 BC", "100 BC:1 AD", "1 AD:150 AD", "150 AD:200 AD"),
```

Comment:
These are consecutive periods. Is this the best display of historical/temporal uncertainty?